README.md

```diff
@@ -69,125 +50,83 @@ RSpec.configure do |c|
 end
 ```
 
-#### manifest\_dir
-Type   | Default  | Puppet Version(s)
------- | -------- | -----------------
-String | Required | 2.x, 3.x
-
-The path to the directory containing your basic manifests like `site.pp`.
-
 #### module\_path
 Type   | Default  | Puppet Version(s)
 ------ | -------- | ------------------
-String | Required | 2.x, 3.x, 4.x, 5.x
+String | Required | any
 
 The path to the directory containing your Puppet modules.
 
 #### default\_facts
 Type | Default | Puppet Version(s)
 ---- | ------- | ------------------
-Hash | `{}`    | 2.x, 3.x, 4.x, 5.x
+Hash | `{}`    | any
 
 A hash of default facts that should be used for all the tests.
 
 #### hiera\_config
 Type   | Default       | Puppet Version(s)
 ------ | ------------- | -----------------
-String | `"/dev/null"` | 3.x, 4.x, 5.x
+String | `"/dev/null"` | any
 
 The path to your `hiera.yaml` file (if used).
 
 #### default\_node\_params
 Type | Default | Puppet Version(s)
 ---- | ------- | -----------------
-Hash | `{}`    | 4.x, 5.x
+Hash | `{}`    | any
 
 A hash of default node parameters that should be used for all the tests.
 
 #### default\_trusted\_facts
 Type | Default | Puppet Version(s)
 ---- | ------- | -----------------
-Hash | `{}`    | 4.x, 5.x
+Hash | `{}`    | any
 
 A hash of default trusted facts that should be used for all the tests
-(available in the manifests as the `$trusted` hash). In order to use this, the
-`trusted_node_data` setting must be set to `true`.
+(available in the manifests as the `$trusted` hash).
 
 #### trusted\_node\_data
 Type    | Default | Puppet Version(s)
 ------- | ------- | -----------------
-Boolean | `false` | >=3.4, 4.x, 5.x
+Boolean | `false` | any
 
 Configures rspec-puppet to use the `$trusted` hash when compiling the
 catalogues.
 
 #### trusted\_server\_facts
 Type    | Default | Puppet Version(s)
 ------- | ------- | -----------------
-Boolean | `false` | >=4.3, 5.x
+Boolean | `false` | any
 
 Configures rspec-puppet to use the `$server_facts` hash when compiling the
 catalogues.
 
 #### confdir
 Type   | Default         | Puppet Version(s)
 ------ | --------------- | ------------------
-String | `"/etc/puppet"` | 2.x, 3.x, 4.x, 5.x
+String | `"/etc/puppet"` | any
 
 The path to the main Puppet configuration directory.
 
 #### config
 Type   | Default                | Puppet Version(s)
 ------ | ---------------------- | ------------------
-String | Puppet's default value | 2.x, 3.x, 4.x, 5.x
+String | Puppet's default value | any
 
 The path to `puppet.conf`.
 
-#### manifest
-Type   | Default                | Puppet Version(s)
------- | ---------------------- | -----------------
-String | Puppet's default value | 2.x, 3.x
-
-The entry-point manifest for Puppet, usually `$manifest_dir/site.pp`.
```

Comment:
It looks like we decided to keep the `manifest` setting so we should probably keep some sort of documentation. Although I'm not sure this documentation is exactly correct. Currently, it seems like the `manifest` setting is used by rspec-puppet to prepend Puppet code to test:
https://github.com/puppetlabs/rspec-puppet/blob/ef01d40601ee3ed967073108e0cfc58002dd6d5c/lib/rspec-puppet/support.rb#L23-L25
https://github.com/puppetlabs/rspec-puppet/blob/ef01d40601ee3ed967073108e0cfc58002dd6d5c/lib/rspec-puppet/support.rb#L124-L137

Reply:
nice spot @nabertrand - totally missed that.
Will update.